app/views/CreditAndRefunds.scala.html

```diff
@@ -253,25 +253,37 @@ <h2 class="govuk-heading-m">
             }
 
             <ul class="govuk-list">
-                <li>
-                    @p(){
-                        @messages("credit-and-refund.bullet-one-prt-1")
-                        @link(link = getWhatYouOwe(isAgent),
-                            messageKey = "credit-and-refund.bullet-one-link",
-                            target = None)
-                        @messages("credit-and-refund.bullet-one-prt-2")
-                    }
-                </li>
+                @if(isAgent) {
+                    <li>
```

Comment:
lines 256-276: the only difference is line 259, can we refactor this code and apply if/else statement in the line 259 instead ?